Fixed CLI logging

nvoxland opened the pull request:
CORE-3220
DAT-3353

nvoxland committed:
Fixed CLI logging

CORE-3220
DAT-3353

SteveDonie reviewed (commented):
Unclear what is happening here. I see a whole class deleted, big block of new import statements, new methods, etc. Would like to get an overview of what the issue was, how it was addressed.

SteveDonie commented:
Ignore my previous comment. Just saw the extensive email from Nathan.

Commit:
DAT-3353: Added `logFile` and `logLevel` to known options to avoid issue with
unexpected argument. Added switch option between `stdout` and `stderr` for
setup messages.

nvoxland commented on liquibase-core/src/main/java/liquibase/integration/commandline/Main.java:
```diff
@@ -390,6 +390,30 @@ protected void reconfigureLogging() {
+        return COMMANDS.SNAPSHOT.equalsIgnoreCase(command)
+                || COMMANDS.EXECUTE_SQL.equalsIgnoreCase(command)
+                || COMMANDS.SNAPSHOT_REFERENCE.equalsIgnoreCase(command)
+                || COMMANDS.UNEXPECTED_CHANGESETS.equalsIgnoreCase(command)
```
I don't think unexpectedChangesets and executeSql should be in this list.

Commit:
DAT-3353: Removed `executeSql`, `unexpectedChangeSets` from list of command
which requires stdout

nvoxland committed:
Reverted changes to expected option flag order validation
- logFile and logLevel go before the command, not after it

CORE-3220
DAT-3353
